packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -37,7 +38,7 @@ import { Math } from "@openzeppelin/contracts/math/Math.sol";
  * Compiler used: solc
  * Runtime target: EVM
  */
-contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
+contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver, Lib_DefaultValues {
```

Comment:
IMO instead of using `is Lib_DefaultValues` it would be better to do `Lib_DefaultValues.DEFAULT_ADDRESS`. It's a little more explicit.